src/core/parse.js

```diff
@@ -34,10 +21,22 @@ export const parse = (obj, paren, wrapper) => {
             if (/&/g.test(key)) next = key.replace(/&/g, paren);
 
             // Media queries or other
-            if (key[0] == '@') next = paren;
-
-            // Call the parse for this block
-            blocks += parse(val, next, next == paren ? key : wrapper || '');
+            if (key[0] == '@') {
+                next = paren;
+                // If this is the case for `@font-face`
+                if (key[1] == 'f') {
+                    next = key;
+                }
+            }
+
+            // If this is the `@keyframes`
+            if (/@k/.test(key)) {
```

Comment:
Just seeing this, make me wonder if I can inline them even more 🤔 cause that would make the `parse` call for `@` rules really specific.

```js
if (key[0] == '@') {
    if (key[1] == 'f') {
    }
    else if (key[1] == 'k') {
  // and so on
    } else {
       blocks += parse(val, next, key);
    }
} else {
    blocks += parse(val, next, wrapper || '');
}
```